### PR TITLE
fix(html): Update PHPDoc in `src` and add new tests for `on*` attribute.

### PR DESCRIPTION
**Pull request by terabytesoftw**
# Pull Request

| Q            | A                                                                  |
| ------------ | ------------------------------------------------------------------ |
| Is bugfix?   | ✔️                                                              |
| New feature? | ❌                                                              |
| Breaks BC?   | ❌                                                              |

### CODE REVIEW

File: tests/Flow/DivTest.php [hunk 1]
```diff
@@ -152,6 +152,20 @@ public function testRenderWithAddDataAttributeUsingEnum(): void
...
+            <<<HTML
+            <div onclick="alert(&apos;Clicked!&apos;)">
+            </div>
+            HTML,
+            Div::tag()
+                ->addEvent('click', "alert('Clicked!')")
+                ->render(),
+            "Failed asserting that element renders correctly with 'addEvent()' method.",
+        );
+    }
```
[the remark this answers is not in the document]
Comment (terabytesoftw) [reply tests/Flow/DivTest.php:167]: It's a test with htmlentitys, and it codes perfectly; the edge case tests are in its repository https://github.com/ui-awesome/html-helper/blob/main/tests/Provider/EncodeProvider.php :)
